profile-submission.json

```
@@ -1,6 +1,9 @@
 {
   "team_profiles": [
     {
+      "github_handle": "abuzaforfagun",
```

Biome (2.1.2)

[error] The key github_handle was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)